test/tests/transfers/multiapp_reporter_transfer/dist_vector/main_rec.i

```diff
@@ -0,0 +1,63 @@
+[Mesh]
+  [generate]
+    type = GeneratedMeshGenerator
+    nx = 4
+    dim = 1
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  solve = false
+[]
+
+[Reporters]
+  [from_sub_rep]
+    type = ConstantReporter
+    real_vector_vector_names = 'rec_vec_vec'
+    real_vector_vector_values = '10000000.'
+    real_vector_names = 'rec_vec'
+    real_vector_values = "0."
+    execute_on = INITIAL
+    outputs = out
+  []
+[]
+
+[Positions]
+  [elem]
+    type = ElementCentroidPositions
+    auto_sort = true
+    outputs = none
+  []
+[]
+
+[MultiApps]
+  [sub]
+    type = FullSolveMultiApp
+    input_files = 'sub.i'
+    positions_objects = elem
+    execute_on = TIMESTEP_END
```

Comment:
pass a cli_arg like 

```
cli_args=Reporters/from_main_rep/real_vector_values=0;Reporters/from_main_rep/real_values=0 
                Reporters/from_main_rep/real_vector_values=1;Reporters/from_main_rep/real_values=-1
                Reporters/from_main_rep/real_vector_values=2;Reporters/from_main_rep/real_values=-2
...
```
so it's not identically 0 0 0 0 and -1 -1 -1 0-1

Reply:
Nice, added that. Now we can see the gather operation better in the test.